source/configuration_manual/fts/flatcurve.rst

```diff
@@ -0,0 +1,168 @@
+.. _fts_backend_flatcurve:
+
+Flatcurve FTS Engine
+====================
+
+.. versionadded:: 2.4.0;3.0.0
+
+Requires Xapian 1.4+ to work properly.
+
+This is a Dovecot FTS plugin to enable message indexing using the
+[Xapian](https://xapian.org/) Open Source Search Engine Library.
+
+The plugin relies on Dovecot to do the necessary stemming. It is intended
+to act as a simple interface to the Xapian storage/search query
+functionality.
+
+This driver supports match scoring and substring matches (on by default),
+which means it is RFC 3501 (IMAP4rev1) compliant. This driver does not
+support fuzzy searches.
+
+The driver passes all of the ImapTest search tests.
+`<https://imapwiki.org/ImapTest>`_
+
+Libraries
+---------
+
+* Xapian: Get ``xapian-core`` v1.4.18 at `<https://xapian.org/download/>`_
+
+Optional parameters
+-------------------
+for the `fts_flatcurve` plugin setting:
```

Comment:
It may have become lost from conversion from github, but I think we should put a strong statement (i.e. in ".. note:" directive or in bold) that the default settings should be fine for most people.  Enabling flatcurve for most people was designed to really be as easy as adding 2 lines, so should add that as an example:

```
mail_plugins = $mail_plugins fts fts_flatcurve
plugin {
  fts = flatcurve
}
```